shardingsphere-mode/shardingsphere-mode-core/src/main/java/org/apache/shardingsphere/mode/metadata/MetaDataContextsBuilder.java

```diff
@@ -46,13 +46,15 @@
  */
 public final class MetaDataContextsBuilder {
     
+    @Getter
     private final Map<String, SchemaConfiguration> schemaConfigMap = new LinkedHashMap<>();
     
     private final Map<String, Collection<ShardingSphereRule>> schemaRulesMap = new LinkedHashMap<>();
     
     @Getter
     private final Map<String, ShardingSphereSchema> schemaMap = new LinkedHashMap<>();
     
+    @Getter
```

Comment:
Why need getter for `globalRuleConfigs`? it can be get from `ContextManagerBuilderParameter` driectly.

Reply:
When the Narayan file is generated, the final transaction configuration needs to be obtained. The configuration in the `server.yaml' file is obtained in the ContextManagerBuilderParameter, not the final configuration.

Reply:
The `globalRuleConfigs` does not change in MetaDataContextsBuilder, is it?